.github/workflows/merge-main.yml

```diff
@@ -55,7 +55,7 @@ jobs:
             file: api/openshift.deploy.yml
             overwrite: true
             parameters:
-              -p OC_NAMESPACE=${{ vars.OC_NAMESPACE }}
+              -p OC_NAMESPACE=a4b31c-test
```

Comment:
Hi Derek, do we also need to do the same `-p OC_NAMESPACE=a4b31c-test` for `merge-demo.yml`?
It looks reasonable to use plain value for `OC_NAMESPACE`; but wonder why `vars.OC_NAMESPACE` has different environment (`-dev`)

Reply:
@ianliuwk1019 You're right about demo!  Change pushed.

GitHub Actions has some weird behaviour where parameters don't use environment overrides.  It's why DEV, which uses vanilla values, doesn't have to be hard coded.